tools/azsdk-cli/docs/cli-commands-guidelines.md

```diff
@@ -90,9 +90,9 @@ For release planning and SDK coordination:
 
 ### 3. **typespec** - TypeSpec Operations and TypeSpec Client Operations
```

Comment:
I think there are some gaps breaking the overall flow between `tsp` and `spec-workflow` commands. From tehe divided command structure it seems that the users working with TypeSpec must use tsp commands for project setup and validation, then switch to spec-workflow commands for generating the SDK and handling the output. 

Should we consider merging the `spec-workflow` functionality into the `tsp` group. This would create a single azsdk tsp command group encompassing the entire TypeSpec workflow, from project initiation to SDK generation. For example, `azsdk tsp generate` could replace `spec-workflow generate-sdk`, and give a continuous task flow?